ci/run-manuscript-editor.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
I think it'd be best to move this to the python manubot package, along with the [other subcommands](https://github.com/manubot/manubot/blob/82f25cb8d698a12d69e533686867a8a4afd35c4c/manubot/command.py#L30-L32).